src/Parsing/Standard/BooleanField.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Mindee\Parsing\Standard;
+
+/**
+ * A field containing a boolean value.
+ */
+class BooleanField extends BaseField
+{
+    use FieldPositionMixin;
+    use FieldConfidenceMixin;
+
+    /**
+     * @var string|null Value as string.
+     */
+    public $value;
+
+
+    /**
+     * @param array        $rawPrediction Raw prediction array.
+     * @param integer|null $pageId        Page number for multi pages document.
+     * @param boolean      $reconstructed Whether the field was reconstructed.
+     * @param string       $valueKey      Key to use for the value.
+     */
+    public function __construct(
+        array $rawPrediction,
+        ?int $pageId = null,
+        bool $reconstructed = false,
+        string $valueKey = 'value'
+    ) {
+        parent::__construct($rawPrediction, $pageId, $reconstructed, $valueKey);
+        $this->setPosition($rawPrediction);
+    }
+
+
+    /**
+     * @return string String representation.
+     */
+    public function __toString(): string
+    {
+        if ($this->value === true) {
+            return 'True';
+        } elseif ($this->value === false) {
+            return 'False';
+        }
+        return "";
```

Comment:
would be better to use the static method in summaryhelper, so only need to fix/modify in one place